TranslationNotifierGenerator/TranslationNotifierGenerator.cs

```diff
@@ -0,0 +1,153 @@
+﻿using System;
+//// uncomment to debug the generator
```

Comment:
You **_really_** wanted to write a comment there, huh?

Reply:
Hah I just `Ctrl+/` another line :P

Reply:
Though I am kinda unsure, my point with it was that someone may accidentally capture the comment when selecting multiple lines with a mouse and uncomment what shouldn't be uncommented, so that was kinda a safeguard against that.